sdk/keyvault/keyvault-certificates/test/CRUD.test.ts

```diff
@@ -59,35 +59,27 @@ describe("Certificates client - create, read, update and delete", () => {
     const controller = new AbortController();
 
     await assertThrowsAbortError(async () => {
-      const poller = await client.beginCreateCertificate(
-        certificateName,
-        basicCertificatePolicy,
-        {
-          ...testPollerProperties,
-          abortSignal: controller.signal
-        }
-      );
+      const poller = await client.beginCreateCertificate(certificateName, basicCertificatePolicy, {
+        ...testPollerProperties,
+        abortSignal: controller.signal
+      });
       controller.abort();
       await poller.pollUntilDone();
     });
   });
 
-  if (isNode && !isPlayingBack) {
+  if (isNode && !isPlaybackMode()) {
     // On playback mode, the tests happen too fast for the timeout to work
```

Comment:
I bet we could fake out some delays to fix this, but let's not worry in this PR

Reply:
Related issue - https://github.com/Azure/azure-sdk-for-js/issues/5747

Reply:
Related issue if you are interested in it - https://github.com/Azure/azure-sdk-for-js/issues/5747

Reply:
Related issue if you are interested in it - https://github.com/Azure/azure-sdk-for-js/issues/5747